src/Kirel.Identity.Server.API/Controllers/RegistrationController.cs

```diff
@@ -7,8 +8,11 @@
 namespace Kirel.Identity.Server.API.Controllers;
 
 /// <inheritdoc />
+
 [ApiController]
 [Route("registration")]
+[EnableIfSetting(true)] 
+[ApiExplorerSettings(IgnoreApi = true)]
```

Comment:
Why we ignore this cantroller always in swagger? It's not correct, if we enable this controller in settings, we should show swagger documentation for him.